packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -418,6 +420,13 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with AutomaticK
   FocusNode _focusNode;
   FocusNode get _effectiveFocusNode => widget.focusNode ?? (_focusNode ??= FocusNode());
 
+  // Is shortly after a previous single tap when not null.
+  Timer _doubleTapTimer;
```

Comment:
Maybe at some point in the future we should factor this into a reusable something (like a gesture recognizer) that just handled tap up and down and cancel events and mapped them to a double tap (or multitap) callback.

Reply:
Perhaps, but having a single long recognizer instead of multiple small discreet recognizers means that one recognizer needs to have hooks to call back all tap/long tap/force tap/drag combinations etc since none of them are second class citizen enough to not be included in a recognizer that just takes in taps up/down. 

Leaving it scenario specific for now until we see other scenarios using something of this complexity.

Reply:
Definitely. Didn't mean to imply that anything needed to be done now.